configure.ac

```diff
@@ -283,11 +283,19 @@ case "${host}" in
 	abi="elf"
 	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
 	;;
-  *-*-openbsd*|*-*-bitrig*)
+  *-*-openbsd*)
 	CFLAGS="$CFLAGS"
 	abi="elf"
 	AC_DEFINE([JEMALLOC_PURGE_MADVISE_FREE], [ ])
+	force_tls="0"
+	sbrk_deprecated="1"
```

Comment:
- autodetection found `sbrk` under openbsd
- the manpage of `sbrk` under openbsd said: 
```
The brk() and sbrk() functions are historical curiosities left over from earlier days before the advent of virtual memory management.
```

it could be seen as a form a deprecation :)

- I don't have any error message with or without `sbrk`
- the `make check` terminate without failure (with or without `sbrk_deprecated="1"`)

so having it forced to be disabled or not under openbsd isn't in my competence... I could just say that it don't introduce regression...